File: Exercise/Medium/02. Sum of List/Code/Sum of List.cs
using System;

class Program
{
    static void SumList()
    {
        string line = Console.ReadLine();
        string[] numbers = line.Split(' ');

        int sum = 0;
        foreach (string num in numbers)
        {
            sum += int.Parse(num);
        }

        Console.WriteLine(sum);
    }

    static void Main()
    {
        SumList();
    }
}

File: Exercise/Medium/02. Sum of List/Code/Sum of List.java
import java.util.Scanner;

public class Main {
    static void sumList() {
        Scanner scanner = new Scanner(System.in);
        String line = scanner.nextLine();
        
        int sum = 0;
        String[] numbers = line.split(" ");
        for (String numStr : numbers) {
            sum += Integer.parseInt(numStr);
        }
        
        System.out.println(sum);
        scanner.close();
    }

    public static void main(String[] args) {
        sumList();
    }
}



